### DISCUSSION TITLE
img2vid

### DISCUSSION THREAD

**Discussion by beepbeepplop** [deforum-art/sd-webui-text2video#160] [General]
Been using this recently to create all kinds of videos. Great stuff. But the img2vid. I don't get it. Can it animate an existing image (as a start frame)? All i can seem to do is fade between the start picture and the video it wants to create. Any tips? Thanks.

**Comment by Northloop** [comment]
I've also been trying to get this to work with no luck. Not much documentation on how to use this out there...
